tests/beaker/tests/create_lv_with_param_extents.rb

```diff
@@ -0,0 +1,48 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96575 - create logical volume with parameter 'extents'"
+
+#initilize
+pv = '/dev/sdc'
+vg = ("VolumeGroup_" + SecureRandom.hex(2))
+lv = ("LogicalVolume_" + SecureRandom.hex(3))
```

Comment:
What does wrapping the value in parenthesis do in Ruby? (I'm not familiar with this syntax.)